fsi-workflow/04_fluidSimulation/tasks.md

```diff
@@ -39,25 +39,25 @@ Open the file `U` and:
 - substitute **UINF** at line 19 with the value **0.5**. This initializes the whole domain to $U_{\infty}$
 - substitute the boundary condition **BOUNDARY** for the *naca2312* patch at line **28** with **noSlip**
 
-Note: we use the folder `0.orig` instead of the usual folder `0` just in case the simulation overwrites the initial conditions (e.g. you perform `potentialFoam` to initialize the fluid domain). The launch script that we prepared will take care of copying `0.orig` to `0`.
+Note: we use the folder `0.orig` instead of the usual folder `0` just in case the simulation overwrites the initial conditions (e.g., you execute `potentialFoam` to initialize the fluid domain). The launch script that we prepared will take care of copying `0.orig` to `0`.
 
 ### `constant` folder
 
 Here you need to perform the following activities:
 
-- Use the **mesh** you generated in the previous task: copy the `polyMesh` folder, that you find in the `0.003` folder, in here
+- Use the **mesh** you generated in the previous task: copy the `polyMesh` folder, which you can find in the `0.003` folder, in here
 - Open the `transportProperties` file to define the kinematic viscosity $\nu$: substitute **NU** at line **19** with `1e-06`
 - Open the `turbulenceProperties` file to define the type of simulation: uncomment line **17** to perform a **laminar** simulation
 
 ### `system` folder
 
 Here you will define how many simulation steps you want to perform and you will make use of *function objects* in order to compute **forces, moments** and **force** and **moment coefficients**:
 
-- Open the `controlDict` file and:
+- Open the `controlDict` file and orientate yourself on the different sections. Then:
     1. substitute **END** with **500** at line **26**: we will perform 500 simulation steps at most
     2. substitute **RHO** with **1000.0** at lines **77** and **102**
     3. substitute **UINF** with **0.2** at line **111**
-    4. substitute **CHORD** with **0.1**at line **112**
+    4. substitute **CHORD** with **0.1** at line **112**
```

Comment:
Mentioning concrete lines is helpful, but tricky to maintain.
Also in this case, I would like to remove the misleading copyright headers from the config files, but then we need to update the numbers as well (I can do that).

Reply:
You are right, I'll remove the line numbers as in the previous tasks. And I'll remove the headers. I have checked here and there but I left some out.